internal/service/dns_config/api_auth_zone_resource_test.go

```diff
@@ -839,7 +839,7 @@ func testAccAuthZoneTags(fqdn string, primaryType string, tags map[string]string
 resource "bloxone_dns_auth_zone" "test_tags" {
     fqdn = %q
     primary_type = %q
-    tags = %s
+    tags = %q
```

Comment:
should have been `%s`, why change to `%q`?

Reply:
Yikes. It should be %s. Maybe I updated it by mistake. Will fix it